diff -ruN docutils/parsers/rst/directives/images.py docutils_new/parsers/rst/directives/images.py
--- docutils/parsers/rst/directives/images.py	2014-06-24 16:52:59.000000000 +0200
+++ docutils_new/parsers/rst/directives/images.py	2017-06-22 09:31:39.057920645 +0200
@@ -48,7 +48,8 @@
                    'align': align,
                    'name': directives.unchanged,
                    'target': directives.unchanged_required,
-                   'class': directives.class_option}
+                   'class': directives.class_option,
+                   'scrollable': directives.flag }
 
     def run(self):
         if 'align' in self.options:
diff -ruN docutils/writers/_html_base.py docutils_new/writers/_html_base.py
--- docutils/writers/_html_base.py	2016-11-29 13:00:39.000000000 +0100
+++ docutils_new/writers/_html_base.py	2017-06-22 09:59:41.605160292 +0200
@@ -892,6 +892,10 @@
             suffix = '\n'
         if 'align' in node:
             atts['class'] = 'align-%s' % node['align']
+        if 'scrollable' in node:
+            # Scrollable flag was given - embed the image tag in a `div` tag
+            # that allows for overflow
+            self.body.append('<div style="overflow: auto; width: auto; height: auto;">\n')
         if ext in self.object_image_types:
             # do NOT use an empty tag: incorrect rendering in browsers
             self.body.append(self.starttag(node, 'object', suffix, **atts) +
@@ -900,8 +904,9 @@
             self.body.append(self.emptytag(node, 'img', suffix, **atts))
 
     def depart_image(self, node):
-        # self.body.append(self.context.pop())
-        pass
+        # close the `div` tag if we opened one
+        if 'scrollable' in node:
+            self.body.append('</div>\n')
 
     def visit_inline(self, node):
         self.body.append(self.starttag(node, 'span', ''))
